src/TransactionTest.java

```diff
@@ -0,0 +1,77 @@
+import org.junit.Test;
+
+import java.util.Date;
+
+import static org.junit.Assert.*;
+
+public class TransactionTest {
+
+    @Test
+    public void givenNoDataOrMoney_returnsNoDataOrMoney() {
+        String summary = new Transaction().getSummary();
+        long money = new Transaction().getMoney();
+        String description = new Transaction().getDescription();
+        assertEquals("No data.", summary);
+        assertEquals(0, money);
+        assertNull(null,description);
+    }
+
+    @Test
+    public void givenMoney_returnsDescriptionAndValue() {
+        String summary = makeTransaction().getSummary();
+        long money = makeTransaction().getMoney();
+        assertEquals("Your account has been credited with £10.00", summary);
+        assertEquals(1000, money);
+    }
+
+    @Test
+    public void givenNegativeMoney_returnsNegativeDescriptionAndValue() {
+        String summary = makeNegativeTransaction().getSummary();
+        String summary2 = makeNegativeTransaction2().getSummary();
```

Comment:
RED: This should perhaps be two separate tests:
`givenRoundNegativeMoney_returnsRoundDetails`
`givenDecimalNegativeMoney_returnsDecimalDetails`